homeassistant/components/geonetnz_quakes/sensor.py

```diff
@@ -68,6 +69,9 @@ def __init__(self, config_entry_id, config_unique_id, config_title, manager):
         """Call when entity will be removed from hass."""
         if self._remove_signal_status:
             self._remove_signal_status()
+        # Remove from entity registry.
+        entity_registry = await async_get_registry(self.hass)
```

Comment:
The sensor entity isn't removed by the integration, right? Its only for that case we need to clean up the entity registry too.

If the user removes the config entry the core will handle clean up of the entity registry.

https://github.com/home-assistant/home-assistant/blob/7ca4665711ca81492903eeb2da2fde9620da7c82/homeassistant/helpers/entity_registry.py#L439-L447

Reply:
Ah, OK, in that case, I don't need to do anything for the sensor because that remains in the system, and I have to move the unit test code to the section that tests removing geolocation entities. Stay tuned...